src/controllers/hotels.controller.js

```diff
@@ -100,11 +102,70 @@ class Hotel {
       type,
       description,
       cost,
-      status
+      status,
     });
 
     return Responses.handleSuccess(201, 'Room added successfully', res, room);
   }
+
+  /**
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} responses
+   */
+  async getMostVisitedDestination(req, res) {
+    const page = (req.query.page * 1) || 1;
```

Comment:
You can define what `1`s mean. They are magical in this context. If it means the first page, then it can be assigned to `firstPage` or something else that is meaningful.

Reply:
I remove this pagination, we will implement it in the Front end if needed